### ISSUE TITLE
Project Page

### ISSUE THREAD

**Issue by djbe** [arranger1044/iSoul#23]
@arranger1044 Could you create a project page? Go to repository settings and there'll be a button to create a (nice) project page (example: http://djbe.github.com/ElectroJam/).

More info about this here: https://github.com/blog/1081-instantly-beautiful-project-pages

**Comment by arranger1044** [comment]
I will look into it right now David, which theme do you prefer?

Il giorno 15/apr/2012, alle ore 20:39, David Jennes ha scritto:

**Comment by djbe** [comment]
Dunno, most of them are quite good, with exception of "Hack", "Minimalist" and "Midnight'.

**Comment by arranger1044** [comment]
Slate is the best one, well it maybe is the only good one, I hope we can customize it later : D

Il giorno 15/apr/2012, alle ore 20:44, David Jennes ha scritto:

**Comment by djbe** [comment]
You can, the project page is an actual github repo (to be more specific, it's a branch of our project with only the html page + related resources).

**Comment by arranger1044** [comment]
Supernice

**Comment by djbe** [comment]
Cool! We might want to change the download links to the actual binaries + add a version number.

**Comment by arranger1044** [comment]
Could you please update them David?
Thanks in advance

Il giorno 15/apr/2012, alle ore 21:02, David Jennes ha scritto:

**Comment by arranger1044** [comment]
I tried to change some spacing and colors to the page, since I am a noob with html&css it maybe disgraceful take a look at it

**Comment by djbe** [comment]
The orange is a bit to strong/glaring, maybe make it a bit darker. Also at the bottom of the page, the blue links on orange? Not very readable.

I'll look into the download buttons when I have some time.

**Comment by arranger1044** [comment]
You are completely right David, I will improve it while learning a bit of html and css since I'm still a newbie at it. Do you have any good book to suggest? : )

Il giorno 17/apr/2012, alle ore 03:18, David Jennes ha scritto:

**Comment by djbe** [comment]
Books for programming languages and such is something I read +-10 years ago, now it's easier to just search for tutorials / sample code / API docs with your favorite search engine. A very good site is: http://www.w3schools.com/ (both for beginners as advanced users).

If you're planning on doing something with JS though, make sure you use a library such as JQuery, otherwise you'll just get nightmares from trying to make your code work in different browsers.
